static/css/announcement.css

```
@@ -1,3 +1,9 @@
+@media print {
+  #announcement, section#announcement {
```

```suggestion
  #fp-announcement, section#fp-announcement {
```

I think this is probably causing the issue I mentioned above.